src/pages/Lists/Lists.vue

```diff
@@ -24,9 +24,20 @@
             :options="{group:{ name:'lists'}}"
             @end="end"
         >
-            <div v-for="({name, games}, index) in lists" :key="name" class="list" >
+            <div v-for="({name, games}, index) in lists" :key="name" class="list">
                 <div class="list-header">
-                    <strong>{{name}}</strong>
+                    <strong
+                        v-show="!editing || editingIndex != index"
+                        @click="startTitleEdit(index, name)"
+                    >
+                        {{name}}
+                    </strong>
+                    <input
+                        v-if="editing && editingIndex == index"
```

Comment:
Since this logic is the exact opposite as `v-show="!editing || editingIndex != index"` you can just use `v-else` here.

Reply:
Fixed!